openpgp/tests/data/contrib/pep/pEpkey-netpgp.asc

```diff
@@ -1,26 +1,26 @@
------BEGIN PGP PUBLIC KEY BLOCK-----

```

Comment:
What changed in this file? They seem similar to me. Maybe new line encoding?

Reply:
Apparently yes, this is automatic